### DISCUSSION TITLE
How to set radiations using Icepak.create_pcb_from_3dlayout() or Icepak.create_ipk_3dcomponent_pcb() ?

### DISCUSSION THREAD

**Discussion by Rungenwagen** [ansys/pyaedt#3479] [Q&A]
In AEDT GUI, there are radiation setting when creating a PCB. Users can switch on radiations on the both sides in the first dialog and set surface materials and "radiate to" in the second dialog.
![AEDT_create_PCB_1](https://github.com/ansys/pyaedt/assets/39006661/5886fd97-c9b1-4854-b468-fc3b810bb398)
![AEDT_create_PCB_2](https://github.com/ansys/pyaedt/assets/39006661/1a9e5571-72c7-47ed-8bdd-bcdbca3c98f3)
But in PyAEDT, the function Icepak.create_pcb_from_3dlayout() can't achieve this. It seems that the function Icepak.create_ipk_3dcomponent_pcb() can only set whether there is radiation on the both sides.

Are the radiation settings already implemented in the functions and how can I use them?
What's the difference between these two functions?

**Comment by Samuelopez-ansys** [comment]
@lorenzovecchietti

**Comment by Samuelopez-ansys** [comment]
@lorenzovecchietti Could you please check this discussion? Thanks

**Comment by lorenzovecchietti** [comment]
Hi Samuel, the option was added in this PR https://github.com/ansys/pyaedt/pull/3788. I mentioned this discussion in the PR, but I see now that the behavior here is different from the issues tab.
